tests/src/test/java/com/maxio/advancedbilling/utils/TestTeardown.java

```diff
@@ -62,7 +62,9 @@ public void deleteSubscriptions() throws IOException, ApiException {
     }
 
     public void deleteSubscriptionGroup(SubscriptionGroupSignupResponse groupSignup) throws IOException, ApiException {
-        deleteSubscriptionGroup(groupSignup.getPrimarySubscriptionId(), groupSignup.getSubscriptionIds(), groupSignup.getCustomerId());
+        if (groupSignup != null) {
```

Comment:
when it's null, does it mean that customer is absent? Because deleteSubscriptionGroup at the end clears customer, so is it possible that we will not clear customer and it will affect other tests?

Reply:
We will not clear anything if groupSignup is null as it will throw NPE.
It's just additional check to prevent NPE in a case there was some exception in test setup.